OpenSim/Examples/ExampleHopperDevice/buildHopperModel.cpp

```diff
@@ -88,15 +88,16 @@ Model buildHopper() {
 
     // Set the coordinate names and default values. Note that we need "auto&"
     // here so that we get a reference to the CoordinateSet rather than a copy.
-    auto& sliderCoord = sliderToGround->upd_CoordinateSet()[0];
+    auto& sliderCoord =
```

Comment:
Hm I'm pretty sure I fixed this already. Anyway, the comment should be updated ("CoordinateSet" -> "Coordinate").